airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/CdcStateCompressionTest.java

```diff
@@ -64,20 +86,32 @@ public void setup() throws Exception {
     // Create a test schema and a bunch of test tables with CDC enabled.
     // Insert one row in each table so that they're not empty.
     testdb.with("CREATE SCHEMA %s;", TEST_SCHEMA);
+    List<Callable<MsSQLTestDatabase>> createAndPopulateTableTasks = new ArrayList<>();
+    List<Callable<MsSQLTestDatabase>> waitForCdcRecordTasks = new ArrayList<>();
+    List<Callable<MsSQLTestDatabase>> alterTabletasks = new ArrayList<>();
+    List<Callable<MsSQLTestDatabase>> enableTableCdctasks = new ArrayList<>();
+    List<Callable<MsSQLTestDatabase>> disableTableCdctasks = new ArrayList<>();
+
     for (int i = 0; i < TEST_TABLES; i++) {
```

Comment:
moved from having 3 loops into a single one that makes more sense. The execution order stays the same, except for multi-threading each step